packages/rum-core/src/common/apm-server.js

```diff
@@ -140,6 +140,29 @@ class ApmServer {
     return new Queue(onFlush, { queueLimit, flushInterval })
   }
 
+  fetchConfig(serviceName, environment) {
+    const serverUrl = this._configService.get('serverUrl')
+    var configEndpoint = `${serverUrl}/config/v1/agents`
+    if (!serviceName) {
```

Comment:
We should not care about validation at this point since its already done when initialising the agent.

Reply:
This function does not have any knowledge about validation process which happens elsewhere, but it needs to handle invalid arguments. In other words if for any reason we pass undefined as serviceName to this function it should be able to handle that instead of making an actual request to the apm server.